gradescopeCronJob/gradescope_to_spreadsheet.py

```diff
@@ -17,27 +17,34 @@
 from google.oauth2.service_account import Credentials
 from dotenv import load_dotenv
 
-
 load_dotenv()
-GRADESCOPE_EMAIL = os.getenv("EMAIL")
-GRADESCOPE_PASSWORD = os.getenv("PASSWORD")
-
-# CS10 Fa24 course id
-# NOTE: Change this `COURSE_ID` variable to change the semester
-COURSE_ID = "831412" #"782967"
-# This scope allows for write access.
-SCOPES = ["https://www.googleapis.com/auth/spreadsheets"]
-SPREADSHEET_ID = "1bqXOiAOYnEUD1FHoK1f_uX4PP3gplRR8maQr3S1zmTQ"
-NUMBER_OF_STUDENTS = 180
+GRADESCOPE_EMAIL = os.getenv("GRADESCOPE_EMAIL")
+GRADESCOPE_PASSWORD = os.getenv("GRADESCOPE_PASSWORD")
+
+# Load JSON variables
+config_path = os.path.join(os.path.dirname(__file__), "config.json")
```

Comment:
Make a folder called "config" and then the files should be in the format `{CLASSNAME}_{SEMESTER}_{YEAR}.json`

Reply:
Hardcode this to load `config/cs10_fall2024.json`